src/rt/vdv/vdv_update.cc

```diff
@@ -0,0 +1,451 @@
+#include "nigiri/rt/vdv/vdv_update.h"
+
+#include <ranges>
+#include <sstream>
+#include <string>
+#include <string_view>
+
+#include "pugixml.hpp"
+
+#include "utl/enumerate.h"
+#include "utl/get_or_create.h"
+#include "utl/parser/arg_parser.h"
+#include "utl/verify.h"
+
+#include "nigiri/routing/for_each_meta.h"
+#include "nigiri/rt/frun.h"
+#include "nigiri/rt/rt_timetable.h"
+#include "nigiri/rt/run.h"
+#include "nigiri/timetable.h"
+#include "nigiri/types.h"
+
+namespace nigiri::rt::vdv {
+
+// #define VDV_DEBUG
+#ifdef VDV_DEBUG
+#define vdv_trace(s) std::cout << s << "\n"
```

Comment:
```suggestion
#define vdv_trace(...) fmt::print(__VA_ARGS__)
```

unten std::format weg